tests/unit/morphing/test_iterable_provider.py

```diff
@@ -46,23 +46,23 @@ def retort():
     )
```

Comment:
Other tests should use Retort, only test_mapping_providing requires own set of providers

Reply:
+